docs/example.rst

```diff
@@ -76,166 +84,179 @@ To do this, we run the following command:
 
-As seen above, the creation of our DataLad dataset is now reflected in the dataset’s version control 
-history. Note that it is best practice to provide a detailed commit message with each change made to
-the data. 
+As seen above,
+the creation of our DataLad dataset is now reflected in the dataset's version control history.
+Note that it is best practice to provide a detailed commit message with each change made to the data.
 
 
 Adding NIfTI Information to JSON Sidecars
--------------------------------------------
+-----------------------------------------
 
 Next, we seek to add more image parameters to our sidecars so that we can better define our Key Groups.
 Historically, only a subset of parameters in the NIfTI image header have been included in a BIDS sidecar...
-Parameters such as image dimensions, number of volumes, image obliquity, and voxel sizes — all important
-data that can change how our pipelines will eventually run!
+Parameters such as image dimensions, number of volumes, image obliquity, and voxel sizes —
+all important data that can change how our pipelines will eventually run!
 
 To add them to the sidecar metadata, run:
 
 .. code-block:: console
 
-    $ cubids-add-nifti-info BIDS_Dataset_DataLad --use-datalad
-
-This command adds the NIfTI header information to the JSON sidecars and saves those changes. In order 
-to ensure that this command has been executed properly, we can run ``cubids-print-metadata-fields`` 
-once more, which reveals that new NIfTI header information has been successfully included in the metadata. 
-Since we ran ``cubids-add-nifti-info`` with the ``--use-datalad`` flag set, ``CuBIDS`` automatically saves
-the changes made to the dataset to the git log as follows:
+    $ cubids add-nifti-info BIDS_Dataset_DataLad --use-datalad
 
+This command adds the NIfTI header information to the JSON sidecars and saves those changes.
+In order to ensure that this command has been executed properly,
+we can run ``cubids print-metadata-fields`` once more,
+which reveals that new NIfTI header information has been successfully included in the metadata.
+Since we ran ``cubids add-nifti-info`` with the ``--use-datalad`` flag set,
+``CuBIDS`` automatically saves the changes made to the dataset to the git log as follows:
 
 .. image:: _static/screenshot_4.png
 
-BIDS validation 
-----------------
+
+BIDS validation
+---------------
 
 The next step in the ``CuBIDS`` workflow is to run BIDS validation
-to detect potential curation errors using ``cubids-validate``.
+to detect potential curation errors using ``cubids validate``.
 
 .. code-block:: console
 
-    $ cubids-validate BIDS_Dataset_DataLad v0 --sequential
+    $ cubids validate BIDS_Dataset_DataLad v0 --sequential
 
-.. note::  The use of the ``--sequential`` flag forces the validator to treat each participant as its own BIDS dataset. This can be helpful for identifying heterogeneous elements, but can be slowed down by extremely large datasets.
+.. note::
```

Comment:
does this still render correctly?

Reply:
Yeah everything still shows up in the admonition box